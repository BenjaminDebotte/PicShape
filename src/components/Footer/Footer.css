.ui.inverted.vertical.fixed.footer.segment {
    position:fixed;
    bottom:0;
    width: 100%;

}
