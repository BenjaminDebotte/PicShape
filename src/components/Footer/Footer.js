import React, { Component } from 'react';
import './Footer.css';

class Footer extends Component {

    render() {
      return (
            <div className="ui inverted vertical fixed footer center aligned segment">
                <span className="text-muted">Â© 2016-2017 Le Conseil, Inc., All Rights Reserved.</span>
            </div>

      );
    }
    }

    export default Footer;
