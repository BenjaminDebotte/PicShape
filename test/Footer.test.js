import React from 'react';
import  {Footer} from '../src/components/Footer/Footer';
import renderer from 'react-test-renderer';

describe('components', function() {
  describe('<Footer />', function() {
    it('renders correctly', function() {
      var tree = renderer.create(<Footer />).toJSON();
      expect(tree).toMatchSnapshot();
        });
  });
});
