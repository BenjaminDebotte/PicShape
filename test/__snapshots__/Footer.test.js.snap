exports[`components <Footer /> renders correctly 1`] = `
<div
  className="ui inverted vertical fixed footer segment">
  <span
    className="text-muted">
    Â© 2016-2017 Le Conseil, Inc., All Rights Reserved.
  </span>
</div>
`;
